packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1807,16 +1822,31 @@ export class Combobox
                   class={CSS.selectAll}
                   id={`${this.guid}-select-all-enabled-interactive`}
                   indeterminate={this.indeterminate}
-                  label={this.messages.selectAll}
+                  label={messages.selectAll}
                   ref={this.selectAllComboboxItemReferenceEl}
                   scale={scale}
                   selected={this.allSelected}
                   tabIndex="-1"
-                  text-label={this.messages.selectAll}
+                  text-label={messages.selectAll}
                   value="select-all"
                 />
               )}
             <slot />
+            {this.noMatches === "add" && this.filterText && (
+              <li
+                aria-label={label}
+                class={CSS.noMatches}
+                onClick={this.customChipAddHandler}
+                role="option"
+                tabIndex={0}
+              >
+                {highlightText({
+                  text: label,
+                  pattern: new RegExp(`(${escapeRegExp(this.filterText)})`, "i"),
+                })}
+              </li>
+            )}
+            {this.noMatches === "none" && <li class={CSS.noMatches}>{messages.noMatches}</li>}
```

Comment:
`this.noMatches === "none"` seems weird to me. What does it mean? Is there anyway to make the state and value a bit clearer?

`noMatches: "add" | "none"| null` isn't totally apparent. Can you think of a better way to describe the behavior?